Makefile

```diff
@@ -8,3 +8,13 @@ default:
 
 refresh-piplock-files:
 	cd caikit && pipenv lock
+
+.PHONY: docker-test
+
+docker-test: default
+	podman run -it --rm \
+	-e DTYPE_STR=float32 \
```

Comment:
Is it possible to move this conf to the template? My understanding is that it is necessary to run on CPU. Correct?

Reply:
It is, but we don't want to set it universally, since GPU workloads may need to set it differently

Reply:
Ok, do you suggest to document that this option is necessary to run on CPU?
Here is a bit hidden in the script

Reply:
It's not needed forever, need is only based on https://github.com/IBM/text-generation-inference/issues/7

Reply:
By the time we're ready for release, should hopefully no longer be necessary